cmd/autopublicizeconfig/main.go

```diff
@@ -96,7 +96,7 @@ type Config struct {
 
 func main() {
 	o := parseOptions()
-	if err := validateOptions(o); err != nil {
+	if err := validateOptions(&o); err != nil {
```

Comment:
I don't understand this change, the `Validation` doesn't seem to mutate anything?

Also if it does, you should rename it to `defaultAndValidate`

Reply:
It mutates the `whitelistOptions`

Reply:
+1 for the name but I should change this in the `whitelist` stuff. I will do this in a follow-up, when I will rename this to `allowlist` as well.

Reply:
Why not change it now? Its not only the `whitelist` stuff its also `validateOptions` because its actually `defaultAndValidateOptions`.